reinforcement_learning/Lesson-4-Policy-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>Might be useful to add a note after the equation picture explaining to the kids that it's ok if they don't recognize or understand what all the symbols are in the equation. Emphasize that the important part is that they recognize that we use an intelligent deep learning approach to learn the optimal policy via gradient descent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>